redwood-treehouse-host/build.gradle

```diff
@@ -58,6 +58,11 @@ kotlin {
         implementation libs.robolectric
       }
     }
+    jvmTest {
+      if (!rootProject.hasProperty('redwoodNoApps')) {
+        kotlin.srcDir('src/appsJvmTest/kotlin')
+      }
+    }
```

Comment:
I mentioned this on the other PR, but these almost certainly aren't going to be run on CI as a result. Need to add the `check` task for this module to the 'test-app' shard.

Reply:
Yep, on it.

Reply:
Done. https://github.com/cashapp/redwood/pull/2065